src/scripts/env.js

```diff
@@ -0,0 +1,90 @@
+// Description:
+//	Displays the version of the bot and all of its external scripts
+//
+// Configuration:
+//
+// Commands:
+//   hubot environment - Shows the current running environment variables
+//   hubot environment <text> - Shows environment variables containing <text> in their name
+//
+// Author:
+//	lanzen
+//
+/*
+ * Licensed Materials - Property of IBM
+ * (C) Copyright IBM Corp. 2016. All Rights Reserved.
+ * US Government Users Restricted Rights - Use, duplication or
+ * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+ */
+'use strict';
+
+const SHOW_ENV = /environment(.*)/i;
+
+var path = require('path');
+var TAG = path.basename(__filename);
+
+// --------------------------------------------------------------
+// i18n (internationalization)
+// It will read from a peer messages.json file.  Later, these
+// messages can be referenced throughout the module.
+// --------------------------------------------------------------
+const i18n = new (require('i18n-2'))({
+	locales: ['en'],
+	extension: '.json',
+	// Add more languages to the list of locales when the files are created.
+	directory: __dirname + '/../messages',
+	defaultLocale: 'en',
+	// Prevent messages file from being overwritten in error conditions (like poor JSON).
+	updateFiles: false
+});
+// At some point we need to toggle this setting based on some user input.
+i18n.setLocale('en');
+
+module.exports = robot => {
+
+	// ------------------------------------------------------------------------
+	// Regex matching entry point.
+	// ------------------------------------------------------------------------
+	robot.respond(SHOW_ENV, (res) => {
```

Comment:
include an `{id: 'hubot.show.environment'}`, this is helpful for debugging purposes.